app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, t(".title")) %>
+<h1><%= t ".title" %></h1>
+<p>
+  <%= t ".descripiton" %>
+  <%= link_to t(".ror"), "https://rubyonrails.org/" %>
+  <%= t ".descripiton_1" %>
```

Comment:
gộp chung lại khai báo 1 biến i18n thôi e, nhưng truyền biến vào cho các link_to nhé, ko tách ra như này sẽ bị sai ngữ pháp hoặc nội dung bị rời rạc
https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations